.github/workflows/push.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           path: priv/plts
           key: ${{runner.os}}-${{matrix.otp}}-${{matrix.elixir}}-plts
 
-      - uses: actions/setup-elixir@v1
+      - uses: erlef/setup-elixir@v1
```

Comment:
`ERLEF` doesn't really scan as an acronym to me yet, lol.

Reply:
Yeah, I have to keep checking that I am typing the correct letters!